tests/acceptance/test_acceptance/test_config.py

```diff
@@ -7,121 +7,322 @@
 from tests.acceptance.helpers import create_and_validate_request_and_response
 
 expected_config = """{
-    "revision": "111",
-    "experimentsMap": {
-        "ab_test1": {
-            "id": "16911963060",
-            "key": "ab_test1",
-            "variationsMap": {
-                "variation_1": {
-                    "id": "16905941566",
-                    "key": "variation_1",
-                    "featureEnabled": false,
-                    "variablesMap": {}
-                },
-                "variation_2": {
-                    "id": "16927770169",
-                    "key": "variation_2",
-                    "featureEnabled": false,
-                    "variablesMap": {}
-                }
-            }
+  "environmentKey": "",
+  "sdkKey": "",
```

Comment:
Hmm, here we're getting empty values. I know why, it's because it's already an existing project and the flag wasn't turned on. I wonder if we should modernize our project for Agent acceptance tests. Currently it uses legacy UI and doesn't include sdk and env keys.
maybe I can create a Jira ticket. And then we update this test so we will have sdk and environment key values in.
What do you think @msohailhussain

Reply:
As a minimum this test works, the keys show, but with empty values. At least the keys can be verified.